apps/frontend/src/app/5_pages/AavePage/components/BorrowAssetsList/components/BorrowModal/BorrowForm.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+
+import { t } from 'i18next';
+
+import {
+  AmountInput,
+  Button,
+  Checkbox,
+  ErrorBadge,
+  ErrorLevel,
+  HealthBar,
+  Link,
+  Paragraph,
+  ParagraphSize,
+  Select,
+  SimpleTable,
+  SimpleTableRow,
+} from '@sovryn/ui';
+import { Decimal } from '@sovryn/utils';
+
+import { AmountRenderer } from '../../../../../../2_molecules/AmountRenderer/AmountRenderer';
+import { AssetRenderer } from '../../../../../../2_molecules/AssetRenderer/AssetRenderer';
+import { useDecimalAmountInput } from '../../../../../../../hooks/useDecimalAmountInput';
+import { translations } from '../../../../../../../locales/i18n';
+import { getCollateralRatioThresholds } from './BorrowForm.utils';
+
+const pageTranslations = translations.aavePage;
+
+type BorrowFormProps = {
+  onSuccess: () => unknown;
+};
+
+export const BorrowForm: FC<BorrowFormProps> = () => {
+  const assetPrice = 3258.47; // TODO: this is mocked data. Replace with proper hook
+  const totalBorrow = Decimal.from(10); // TODO: this is mocked data. Replace with proper hook
+  const collateralToLoanRate = Decimal.from(10); // TODO: this is mocked data. Replace with proper hook
+  const collateralSize = Decimal.from(10); // TODO: this is mockd data. Replace with proper hook
+  const borrowableAssets = useMemo(() => ['BTC', 'SOV'], []); // TODO: this is mocked data. Replace with proper hook
```

Comment:
useless memo. Even you could define it outside the component if you prefer to avoid redeclarations

Reply:
Well totally agree! I was just resolving linting and as this is mocked data, thought didn't matter much, but I can change it sure ))